mqtt/mqtt-bridge/tests/config.json

```diff
@@ -4,11 +4,15 @@
     "moduleid": "mymodule",
     "modulegenerationid": "321",
```

Comment:
What casing are/should we use for these fields.

Reply:
it's a string